### PR TITLE
Add Transaction FE

### PR DESCRIPTION
**Pull request by pivilartisant**
Screenshot 2023-08-28 at 17 09 35

### CODE REVIEW

**Review by B-Naoufal — APPROVED** [review]

File: wails-frontend/src/pages/PasswordPromptHandler/Sign.tsx [hunk 1]
```diff
@@ -102,6 +103,12 @@ export function Sign() {
                     <BuySellRoll {...signData} />
                   </>
                 );
+              case 'Transaction':
+                return (
+                  <>
+                    <Transaction {...signData} />
+                  </>
+                );
```
Comment (mazmassa) [inline wails-frontend/src/pages/PasswordPromptHandler/Sign.tsx:111]:
```suggestion
                return <Transaction {...signData} />
```

File: wails-frontend/src/pages/PasswordPromptHandler/Sign.tsx [hunk 2]
```diff
@@ -102,6 +103,12 @@ export function Sign() {
...
               case 'Plain Text':
                 return (
                   <>
```
Comment (mazmassa) [inline wails-frontend/src/pages/PasswordPromptHandler/Sign.tsx:114]: Not part of this code added but seems same question here! `<> </>`

File: wails-frontend/src/pages/PasswordPromptHandler/Transaction/Transaction.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+import { PromptRequestData } from '../Sign';
+import Intl from '@/i18n/i18n';
+import { FiArrowRight } from 'react-icons/fi';
+import { WindowSetSize } from '@wailsjs/runtime/runtime';
+
+export function Transaction(props: PromptRequestData) {
+  const { WalletAddress, RecipientAddress, OperationType, Amount } = props;
+
+  WindowSetSize(460, 500);
+  return (
```
Comment (mazmassa) [inline wails-frontend/src/pages/PasswordPromptHandler/Transaction/Transaction.tsx:11]:
```suggestion
  WindowSetSize(460, 500);
  
  return (
```

**Review by mazmassa — APPROVED** [review]
Left some minor comments, otherwise LGTM!

File: wails-frontend/src/pages/PasswordPromptHandler/Sign.tsx (hunk 1 above)
Comment (pivilartisant) [reply wails-frontend/src/pages/PasswordPromptHandler/Sign.tsx:111]: do you want me to apply these changes to each case where it is present ?

Comment (mazmassa) [reply wails-frontend/src/pages/PasswordPromptHandler/Sign.tsx:111]: You can open a MR specific for that in future, no worries. But keep at least in your code the good one. 👼

**Pull request merged**